src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,86 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[] | null;
```

Comment:
The `todos` prop is typed as `Todo[] | null`. It would be more consistent to initialize `todos` as an empty array instead of `null`, as this would simplify the logic and avoid potential null checks when mapping over the todos.